django/conf/global_settings.py

```diff
@@ -436,6 +436,11 @@ def gettext_noop(s):
 # you may be opening yourself up to a security risk.
 SECURE_PROXY_SSL_HEADER = None
 
+# Default hashing algorithm to use for encoding cookies, password reset tokens
+# in the admin site, user sessions, and signatures. Algorithm must be supported
+# by hashlib.
+DEFAULT_HASHING_ALGORITHM = 'sha256'
```

Comment:
The default needs to be sha1 for existing projects. New projects can have the setting in the project template with `sha256`.

Reply:
Does it? This is a specialized setting for people who need to run two versions in parallel. Those people should be able to read the release notes and prepare an upgrade plan. For everyone else (+new projects) `sha256` would be just fine as default.